site/en/docs/extensions/mv3/overview/index.md

```diff
@@ -6,25 +6,79 @@ updated: 2018-02-25
 description: An overview of the purpose of Chrome Extensions and how they're developed.
 ---
 
-Extensions are small software programs that customize the browsing experience. They enable users to
-tailor Chrome functionality and behavior to individual needs or preferences. They are built on web
-technologies such as HTML, JavaScript, and CSS.
+This page provides a very brief introduction to what extensions are and how you can make a trivial one.
 
-An extension must fulfill a [single purpose][1] that is narrowly defined and easy to understand. A
+
+## About extensions
+
+Extensions are small software programs that customize the browsing experience. They let users tailor
+Chrome functionality and behavior in many ways, providing things like:
+
+* Productivity tools
+* Web page content enrichment
+* Information aggregation
+* Fun and games
+
+These are just a few examples of the many things that extensions can do. See the [Chrome Web
+Store][5] to see thousands of different examples of published extensions.
+
+### How do extensions work?
+
+Extensions are built on web technologies such as HTML, JavaScript, and CSS.  They run in a separate,
+sandboxed execution environment and interact with the Chrome browser.  Consider the context for this
+-- Chrome runs on a device, connecting the user to resources on the Web, as depicted in this context
+diagram:
+
+{% Img src="image/SHhb2PDKzXTggPGAYpv8JgR81pX2/5qVlqGn6vVMCWCSRclOY.png", alt="Context diagram of Chrome with an extension", width="517", height="476" %}
```

Comment:
I'm not sure I understand this diagram.  Is the extension a part of Chrome?  Does it not have access to HTML APIs, storage, and UI (which are separate)?  Can it not access the internet?

Can you describe what the main takeaway we want the reader to have from this diagram is?

Reply:
What I'd like to depict is that Chrome does its thing as a browser, and then extensions "plug onto" Chrome and have access (via the API) of selected Chrome features to intercede in the way they work.

Reply:
For things like this I miss whiteboarding in the office...

Reply:
Hmm... Honestly, for now, I wonder if it's better to just omit this.  I don't think, in its current form, the diagram adds much information.  I wonder if just describing this (similar to what you have) would be sufficient?

Extensions are built on web technologies such as HTML, JavaScript, and CSS.  They run in a separate,
sandboxed execution environment and interact with the Chrome browser.  Extensions are designed to "extend" the Chrome browser -- they utilize APIs in order to modify browser behavior and access web content.  There are two aspects to consider:

WDYT?

Reply:
Yeah, I wanted a diagram to a) help break up the slabs of text and b) help construct context. But I am fine with suppressing this for now and trying to come up with something more meaningful later.